app/models/game.rb

```diff
@@ -1,17 +1,21 @@
 class Game < ActiveRecord::Base
   belongs_to :user
   belongs_to :sport
-  belongs_to :state
   has_many :players, dependent: :destroy
 
   validates :user, presence: true
-  validates :description, presence: true,
-    uniqueness: true,
-    length: { maximum: 20}
+  validates :description, uniqueness: true,
+    length: { maximum: 30}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.